scenarioo-client/test/e2e/pages/branchAliasesPage.ts

```diff
@@ -35,13 +35,12 @@ class BranchAliasesPage {
         const rowToEditIndex = count - 1;
         const rowToEdit = rows.get(rowToEditIndex);
 
-        await rowToEdit.$('input[name="aliasName"]').sendKeys(name);
+        await rowToEdit.$('input[data-type="branchAlias"]').sendKeys(name);
         if (referencedBranchName !== '') {
```

Comment:
Maybe this method could be split up in two  "enterAlias" and "enterAliasWithoutBranchReference" or something similar. Would make it even clearer to comprehend in the test imho.

Reply:
done